eePlumB/Z_PrepAOI/validationSet.js

```diff
@@ -0,0 +1,209 @@
+// This script creates the validation collection for eePlumB
+// written by B. Steele
+// 2020-03-07
+
+var aoi1 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_1');
+var aoi2 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_2');
+var aoi3 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_3');
+var aoi4 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_4');
+var aoi5 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_5');
+var aoi6 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_6');
+var aoi7 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_7');
+var aoi8 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_8');
+var aoi9 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_9');
+var aoi10 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_10');
+var aoi11 = ee.FeatureCollection('projects/ee-ross-superior/assets/tiledAOI/SuperiorAOI_11');
+
+//------------------------------------//
+// DATES FOR USER VALIDATION ------------//
+//-------------------------------------//
+var date0 = '2013-07-23';
+var date1 = '2020-08-27';
+var date2 = '2016-11-04';
+var date3 = '2022-10-28';
+var date4 = '2022-04-19';
+
+// notes from b's noodling around - to see these yourself, uncomment
+// the last two blocks of code and fill in dates on line 123, aoi on 128,
+// you don't need to adjust aoi_id for these
+//date0, aoi9 -- cloud artifacts
+//date0, aoi1 -- clear sediment on south shore; cloud artifacts in NE corner
+//date1, aoi1 -- lots of sediment and deep sediment that looks like blooms
+//date1, aoi2 -- lots of sediment and deep sediment that looks like blooms
+//date1, aoi4 -- unmasked clouds, sediment along s shore
```

Comment:
The unmasked clouds are very hard to see, but I suppose that is the point?

![image](https://user-images.githubusercontent.com/13220910/224204555-6caf1c4e-dd8f-4118-a0fb-6ab74742a703.png)

Reply:
I was also zooming in to see these, so agree that perhaps that means the extent is too large. Maybe slightly smaller would do the trick?